SpatialGDK/Documentation/content/whatsnew.md

```diff
@@ -1,25 +1,29 @@
 # What's new in alpha?
 
-If you’ve used the pre-alpha GDK you may be interested in these latest changes.
+If you’ve used the GDK in pre-alpha  you may be interested in the latest changes.
```

Comment:
```suggestion
If you’ve used the GDK in pre-alpha you may be interested in the latest changes.
```

Reply:
Done, ta. Added comma, which is better IMHumbleO.